tasks/main.yml

```diff
@@ -7,11 +7,11 @@
 
 - name: Import removal tasks
   ansible.builtin.import_tasks: removal.yml
-  when: autorestic_state == 'absent'
+  when: autorestic_state is "absent"
```

Comment:
@whysthatso - Did you have some issues with the syntax here? The molecule tests seemed to be passing OK?

Reply:
now it's good. there was a silly directory mistake, i've gotten rid of the intermediary bin. now it's all good.
also, will branch off of main in the future...